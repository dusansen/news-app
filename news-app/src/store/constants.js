export const SET_COUNTRY = 'SET_COUNTRY';
export const SET_ARTICLES = 'SET_ARTICLES';
export const SET_CURRENT_ARTICLE = 'SET_CURRENT_ARTICLE';
export const SET_ERROR_MESSAGE = 'SET_ERROR_MESSAGE';