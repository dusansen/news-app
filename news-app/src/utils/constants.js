export const COUNTRIES = [
  {
    label: 'Great Britain',
    value: 'GB'
  },
  {
    label: 'United States',
    value: 'US'
  }
];

export const TOP_ARTICLES_CATEGORY = 'top';
export const SEARCH_CATEGORY = 'search';

export const CATEGORIES = [
  'entertainment',
  'general',
  'health',
  'science',
  'sports',
  'technology'
];