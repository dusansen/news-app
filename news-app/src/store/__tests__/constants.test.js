import * as constants from '../constants';

describe('Store constants', () => {
  it('should match snapshot', () => {
    expect(constants).toMatchSnapshot();
  });
});