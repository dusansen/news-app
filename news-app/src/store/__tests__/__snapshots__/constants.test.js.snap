// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store constants should match snapshot 1`] = `
Object {
  "SET_ARTICLES": "SET_ARTICLES",
  "SET_COUNTRY": "SET_COUNTRY",
  "SET_CURRENT_ARTICLE": "SET_CURRENT_ARTICLE",
  "SET_ERROR_MESSAGE": "SET_ERROR_MESSAGE",
}
`;
